### DISCUSSION TITLE
docs not list audio service

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#232] [Q&A] [answered]
from mycroft.skills.audioservice import AudioService 

what is the name of the ovos library?  Not listed in the docs

**Comment by JarbasAl** [comment, marked as answer]
If you  share your skill we can give better advice, but the direct replacement is
`from ovos_bus_client.apis.ocp import OCPAudioServiceInterface`

this works a bit differently under OVOS, while in mycroft this was the only way to play audio, in OVOS this is a subsystem of the media handling framework, it's strongly recommended to move to `OCPInterface` instead. 

I'm still working in OCP docs and will post a link here later.
